cpp-client/deephaven/client/src/impl/table_handle_impl.cc

```diff
@@ -547,7 +545,7 @@ class GetColumnDefsCallback final :
 
   void invokeHelper() {
     arrow::flight::FlightCallOptions options;
-    options.headers.push_back(server_->makeBlessing());
+    options.headers.push_back(server_->getAuthHeader());
```

Comment:
We've talked about interceptors a couple of times but because all my calls are (a) async, and (b) go out the same place in the code and come back to the same place, I felt it was easier to add and look at the headers there.

On the sending side it happens in Server::sendRpc (a template function in the file deephaven/client/include/privde/deephaven/client/server/server.h L253)

On the receiving side I pull out auth headers in the method Server::processNextCompletionQueueItem (which lives in deephaven/client/src/server/server.cc L459)

Reply:
Not using interceptors is IMHO much better/cleaner; I wasn't implying the need, just trying to put forward how the java one was done.